pkg/lite-apiserver/options/server_options.go

```diff
@@ -29,13 +27,15 @@ type RunServerOptions struct {
 	KubeApiserverUrl  string
 	KubeApiserverPort int
 	Port              int
-	SyncDuration      int
 	BackendTimeout    int
 	CAFile            string
 	CertFile          string
 	KeyFile           string
 	ApiserverCAFile   string
+	CacheType         string
 	FileCachePath     string
+	BadgerCachePath   string
+	BoltCacheFile     string
 }
 
 func NewRunServerOptions() *RunServerOptions {
```

Comment:
Could define default values in this function.

Reply:
default values define in `AddFlags`